Running-Mastodon/Development-guide.md

```diff
@@ -128,6 +128,23 @@ Go to http://localhost:3000 to see your development instance.
 
 Admin account is `admin@localhost:3000`. Password is `mastodonadmin`.
 
+## Bash on Windows
+
+To run Mastodon on [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about), you can follow the Ubuntu instructions with some small tweaks.
```

Comment:
I think we could okay this if it's specified that running Mastodon on bash on windows is TOTALLY UNSUPPORTED AND NOT RECOMMENDED.